db/version_set.cc

```diff
@@ -1250,6 +1257,7 @@ void Version::Get(const ReadOptions& read_options, const LookupKey& k,
         } else if (fp.GetHitFileLevel() >= 2) {
           RecordTick(db_statistics_, GET_HIT_L2_AND_UP);
         }
+        PERF_COUNTER_BY_LEVEL_ADD(user_key_return_count, 1, fp.GetHitFileLevel());
```

Comment:
I think we need to find it in kMerge too.

Reply:
I think we don't deal with all kMerge case now. It's OK for now if you add a TODO here, and comment in the perf counter what the expectation is.